platform/testdata/opensearch_requests.go

```diff
@@ -171,7 +171,7 @@ var OpensearchSearchTests = []SearchTestCase{
 			"track_total_hits": true
 		}`,
 		WantedSql: []string{
-			`("__timestamp">=fromUnixTimestamp64Milli(1712236698149) AND "__timestamp"<=fromUnixTimestamp64Milli(1712237598149))`,
+			`("-@timestamp">=fromUnixTimestamp64Milli(1712236698149) AND "-@timestamp"<=fromUnixTimestamp64Milli(1712237598149))`,
```

Comment:
it's fine, WantedSql` = before transformation now, `WantedQueries` contains SQL with properly escaped `__timestamp`